### PR TITLE
Styling update to fonts

### PR DESCRIPTION
**Pull request by Ijin08**
Have updated the variables template to use variables from default. Also changed all font-sizes from rem to px. 

- Removed display classes and variables since not used. 
- Removed lead class and variables since not used. 
- Removed serif font since not used and probably never should be used.

### CODE REVIEW

File: packages/grafana-ui/src/themes/_variables.scss.tmpl.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ $enable-hover-media-query: false !default;
 // Control the default styling of most Bootstrap elements by modifying these
 // variables. Mostly focused on spacing.
 
-$spacer: 1rem !default;
+$spacer: .875rem !default;
```
Comment (torkelo) [inline packages/grafana-ui/src/themes/_variables.scss.tmpl.ts]: can we use pixels instead? so much easier to get things to align

File: packages/grafana-ui/src/themes/default.ts [hunk 2]
```diff
@@ -5,23 +5,23 @@ const theme: GrafanaThemeCommons = {
...
       monospace: "Menlo, Monaco, Consolas, 'Courier New', monospace",
     },
     size: {
-      base: '13px',
-      xs: '10px',
-      s: '12px',
-      m: '14px',
-      l: '18px',
+      root: '100%',
+      base: '.8125rem',
```
Comment (torkelo) [inline packages/grafana-ui/src/themes/default.ts]: would prefer pixel base, as it makes things o much easier.

Comment (Ijin08) [reply packages/grafana-ui/src/themes/default.ts]: The problem with pixels is the user can't change the font size. I was also thinking of using rem for margin and padding forcing us to use .25rem, .5rem, .75rem, 1rem, 1.5rem. Making things aligned. I don't think rem will be much harder than pixels.

**Comment by torkelo** [comment]
This looks great, just need some manual testing and it looks good to merge

**Pull request merged**